### PR TITLE
bounceback 1.4.2 (new formula)

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/b/bounceback.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    pkgshare.install "data"
+  end
+
+  test do
+    cp pkgshare/"config.yml", testpath
+    mkdir testpath/"data"
+    cp pkgshare/"data/allowed_ips.txt", testpath/"data/allowed_ips.txt"
+    cp pkgshare/"data/banned_ips.txt", testpath/"data/banned_ips.txt"
+    cp pkgshare/"data/banned_words.txt", testpath/"data/banned_words.txt"
+    cp pkgshare/"data/default.profile", testpath/"data/default.profile"
```
Comment (chenrui333) [inline Formula/b/bounceback.rb:28]:
```suggestion
    cp_r pkgshare/"data", testpath
```

File: Formula/b/bounceback.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    fork do
+      exec bin/"bounceback"
+    end
+    sleep 2
+    assert_path_exists testpath/"bounceback.log"
```
Comment (chenrui333) [inline Formula/b/bounceback.rb:34]: can we assert the log?

File: Formula/b/bounceback.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/bounceback"
+
+    pkgshare.install "config.yml"
+    pkgshare.install "data"
+  end
+
```
Comment (chenrui333) [inline Formula/b/bounceback.rb:21]: it might be better that we can also add `service` block in here. (looks like a server app)

**Comment by chenrui333** [comment]
should be good now

**Review by ZhongRuoyu — APPROVED** [review]

**Pull request merged**